projects/envoy/build.sh

```diff
@@ -61,10 +63,11 @@ bazel build --verbose_failures --dynamic_mode=off --spawn_strategy=standalone \
 # Copy out test binaries from bazel-bin/ and zip up related test corpuses.
 for t in ${FUZZER_TARGETS}
 do
+  CORPUS_TARGET=`python $SRC/find_corpus.py $t`
```

Comment:
I would write this `CORPUS_TARGET=$(python "${SRC}"/find_corpus.py "$t")` to ensure proper escaping.